### PR TITLE
Update circle3Tangents documentation

### PR DESCRIPTION
**Pull request by lbartoletti**
Update documentation of `circle from 3 tangents` after https://github.com/qgis/QGIS/pull/40702

I'm not sure about the wording.

### CODE REVIEW

**Comment by lbartoletti** [comment]
cc @DelazJ

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -1435,7 +1435,9 @@ curved geometry, if not, QGIS will segmentize the circular arcs.
 - |circle3Tangents| :sup:`Add circle from 3 tangents`: Draws a circle that is
   tangential to three segments. **Note that you must activate snapping to
   segments** (See :ref:`snapping_tolerance`). Click on a segment to add a
-  tangent. If two tangents are parallel, an error message appears and the input
+  tangent. If two tangents are parallel, we take the coordinates of the first 
+  click on the first parallel tangent to determine the positioning of the circle.
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:1439]:
```suggestion
  tangent. If two tangents are parallel, the coordinates of the click on the
  first parallel tangent are used to determine the positioning of the circle.
```
too many first ;)

**Review by DelazJ — COMMENTED** [review]
LGTM. Was it backported to 3.16? Or will it be?

**Comment by lbartoletti** [comment]
> LGTM. Was it backported to 3.16? Or will it be? Thanks @lbartoletti

No, I don't think we should backport it. It is both a fix and a feature.

Thanks Harrissou

**Pull request merged**